src/coreclr/vm/fcall.h

```diff
@@ -86,11 +86,6 @@
 //      COMPlusThrow(execpt);
 //      HELPER_METHOD_FRAME_END()
 
-// It is more efficient (in space) to use convenience macro FCTHROW that does
```

Comment:
Nit: The whole paragraph can be deleted - it is unfinished sentence now. (It does not matter a whole lot - I expect the whole file is going to be deleted soon.)